css/main.css

```diff
@@ -16,68 +11,144 @@ pre {
 ul, ol {
     margin-top: 1.5em;
 }
-h1, h2 {
+h1 {
+    margin-top: 1.5em;
+    font-size: 2.3em;
+}
+h2 {
     margin-top: 1.5em;
+    font-size: 2em;
 }
 ul, ol {
     margin-left: 1em;
 }
 
 h1, h2, h3 { font-weight: 300; }
 
+a:link, a:visited {
+    color: #1c6fef;
+}
+
+html {
+    min-height: 100%;
+    margin: 0;
+}
+
 body {
     display: flex;
     flex-direction: column;
     align-items: center;
     min-height: 100vh;
     font-family: "Open Sans", "DejaVu Sans", sans-serif;
+    font-size: 20px;
```

Comment:
What's the current best practice about font-size?

My understanding is that setting the base font size using `px` units is, in theory, wrong: ideally, we should use "default" font-size, which is overrideable in the user agent.

At the same time, few people actually override the default, and the default is small.

Reply:
If we don't set font-size in pixels, I think we should try to avoid pixels in other cases as well, and stick with ch/em.

Reply:
Alright, I'll update it to use `%` in the html tag and `rem` everywhere else.